Tests/ttLib/tables/C_O_L_R_test.py

```diff
@@ -147,12 +147,12 @@ def test_round_trip_xml(self, font):
     (b"\x00\x00\x08", "Offset to BackdropPaint from beginning of PaintComposite (8)"),
     (b"\x0d", "BaseGlyphV1Record[1].Paint.BackdropPaint.Format (13)"),
     (b"\x00\x00\x34", "Offset to Paint from beginning of PaintVarTransform (52)"),
-    (b"\x00\x01\x00\x00\x00\x00\x00\x00", "Affine2x3.xx.value (1.0)"),
-    (b"\x00\x00\x00\x00\x00\x00\x00\x00", "Affine2x3.xy.value (0.0)"),
-    (b"\x00\x00\x00\x00\x00\x00\x00\x00", "Affine2x3.yx.value (0.0)"),
-    (b"\x00\x01\x00\x00\x00\x00\x00\x00", "Affine2x3.yy.value (1.0)"),
-    (b"\x01\x2c\x00\x00\x00\x00\x00\x00", "Affine2x3.dx.value (300.0)"),
-    (b"\x00\x00\x00\x00\x00\x00\x00\x00", "Affine2x3.dy.value (0.0)"),
+    (b"\x00\x01\x00\x00\xFF\xFF\xFF\xFF", "Affine2x3.xx.value (1.0)"),
```

Comment:
nit: I believe the rest of the hex dump is using lowercase letters (like python bytes repr)